scripts/e2e

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+set -e
+
+
+
+cd $(dirname $0)/..
+
+if [ ! -f ./bin/terraform-controller ]; then
+    echo "./bin/terraform-controller not built, make build-controller to create."
```

Comment:
This could just run the build script to make it nicer for the user

Reply:
fixed!